### PR TITLE
Substrate Overview

### PR DESCRIPTION
**Pull request by jboetticher**
### Description

Created a short overview page of Substrate to help with keywords. Feels kind of like a word salad, so feel free to suggest if there are any good topics to add in or cut.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [X] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/build/substrate-api/substrate.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# Polkadot's Substrate on Moonbeam
+
+![Substrate banner](/images/builders/build/substrate-api/substrate/substrate-banner.png)
+
+## Introduction {: #introduction }
+ 
+Polkadot's Substrate is an open-source, modular SDK for building blockchains in Rust. It is developed by Parity Technologies, the same team behind Polkadot.  
+
+Substrate provides the foundational building blocks for creating custom, domain-specific blockchains through its framework FRAME. FRAME is designed to be flexible and forkless, allowing developers to choose the components that best suit their blockchain needs and perform runtime upgrades without requiring network forks like Ethereum. Most developers will gravitate towards Substrate with the intention to integrate with the Polkadot ecosystem, but its flexibility allows users to create both solo-chains and parachains.
```
Comment (eshaben) [inline builders/build/substrate-api/substrate.md:15]: "Polkadot vs Substrate" are keywords that some search for, so it'd be great to explicitly touch on the relationship between polkadot and substrate, which I think you do here, but just more explicitly state "Polkadot does this, Substrate does this"

File: builders/build/substrate-api/.pages [hunk 2]
```diff
@@ -5,3 +5,4 @@ nav:
   - 'Polkadot.js API': 'polkadot-js-api.md'
   - 'Py Substrate Interface': 'py-substrate-interface.md'
   - 'Sidecar': 'sidecar.md'
+  - 'Substrate': 'substrate.md'
```
Comment (eshaben) [inline builders/build/substrate-api/.pages:8]: this should just be an overview page `'Overview': 'overview.md'`

**Review by eshaben — CHANGES_REQUESTED** [review]
I think it'd be cool to add a section on native interoperability with substrate

File: builders/build/substrate-api/substrate.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+title: Polkadot's Substrate in Moonbeam
+description: Discover Polkadot's Substrate developer framework and how it shapes blockchain development in both Moonbeam's networks and other parachains.
+
+---
+
+# Polkadot's Substrate on Moonbeam
+
+![Substrate banner](/images/builders/build/substrate-api/substrate/substrate-banner.png)
+
+## Introduction {: #introduction }
```
[the remark this answers is not in the document]
Comment (jboetticher) [reply builders/build/substrate-api/substrate.md:11]: We can just replace "Introduction" with "What is Substrate?" since those are better keywords.

**Pull request merged**